charts/lotus-fullnode/templates/service-load-balancer.yaml

```diff
@@ -26,3 +26,31 @@ spec:
       targetPort: 1347
       name: libp2p
 {{- end }}
+{{- if .Values.apiLoadBalancer.enabled }}
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}-lotus-daemon-api-lb
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: lotus-fullnode-app
+    release: {{ .Release.Name }}
+{{- with .Values.additionalLabels }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+  annotations:
+{{- with .Values.apiLoadBalancer.annotations }}
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+  type: LoadBalancer
```

Comment:
Why LoadBalancer and not ClusterIP? Is there a reason to expose this to the outside?